database-plugins/src/main/java/io/cdap/plugin/DBUtils.java

```diff
@@ -149,21 +149,21 @@ public static DriverCleanup ensureJDBCDriverIsAvailable(Class<? extends Driver>
    * Given the result set, get the metadata of the result set and return
    * list of {@link io.cdap.cdap.api.data.schema.Schema.Field},
    * where name of the field is same as column name and type of the field is obtained using
-   * {@link DBUtils#getSchema(String, int, int, int, String)}
+   * {@link DBUtils#getSchema(String, int, int, int, String, boolean)}
    *
    * @param resultSet result set of executed query
    * @return list of schema fields
    * @throws SQLException
    */
-  public static List<Schema.Field> getOriginalSchema(ResultSet resultSet) throws SQLException {
-    return getSchemaFields(resultSet, null, null);
+  public static List<Schema.Field> getOriginalSchema(ResultSet resultSet, Schema outputSchema) throws SQLException {
```

Comment:
Mark `Schema outputSchema` as nullable

Reply:
Javadoc should be updated and should have same params as number of params in the method